bugsnag-plugin-android-ndk/src/main/assets/include/event.h

```diff
@@ -184,6 +190,13 @@ double bugsnag_event_get_metadata_double(void *event_ptr, char *section, char *n
 char *bugsnag_event_get_metadata_string(void *event_ptr, char *section, char *name);
 bool bugsnag_event_get_metadata_bool(void *event_ptr, char *section, char *name);
 
+/* Breadcrumbs */
+
+void bugsnag_event_clear_breadcrumbs(void *event_ptr);
+int bugsnag_event_get_breadcrumbs_size(void *event_ptr);
+bsg_breadcrumb_t bugsnag_event_get_breadcrumb(void *event_ptr, int index);
+void bugsnag_event_set_breadcrumb(void *event_ptr, bsg_breadcrumb_t breadcrumb, int index);
```

Comment:
These should return/accept `bsg_breadcrumb_t *`: returning a reference to the actual breadcrumb when getting and performing a copy of the struct when setting.